src/components/post-comments.jsx

```diff
@@ -57,8 +57,12 @@ export default (props) => {
       props.toggleCommenting(props.post.id)
     }
 
-    const updatedCommentText = `@${username} ` + (props.post.newCommentText || '')
-    props.updateCommentingText(props.post.id, updatedCommentText)
+    const text = (props.post.newCommentText || '')
+    const prefix = `@${username}`
+
+    if (text.indexOf(prefix) === -1){
+      props.updateCommentingText(props.post.id, `${prefix} ${text}`)
+    }
```

Comment:
I would change this logic to the following:
* if the text is empty, start text with "@username"
* if the text is not empty, and (if @username is not in the text OR @username is NOT at the end of the text) then append " @username"

meaning that you can for example add several usernames by clicking on their comment clouds one after another.